src/CommunityToolkit.Maui.Core/Views/DrawingView/PlatformView/MauiDrawingView.windows.cs

```diff
@@ -27,7 +27,7 @@ public void Initialize()
 		}
 		else
 		{
-			System.Diagnostics.Trace.WriteLine("DrawingView requires Windows 10.0.18362 or higher.");
+			System.Diagnostics.Trace.TraceError("DrawingView requires Windows 10.0.18362 or higher.");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.